doc/installation.md

```diff
@@ -0,0 +1,48 @@
+# Installation
+
+## Binaries
+
+`isaac_ros2_control` is currently only released for ROS2 Humble and Rolling. For other ROS2 releases, you may need to build it from source.
+
+        sudo apt-get install ros-$ROS_DISTRO-isaac-ros2-control
+
+## Build from Source
```

Comment:
FYI I'm going to make a follow up PR to build a docker container that users can run the demo against.